testing/protovalidate-conformance/expected_failures.yaml

```diff
@@ -0,0 +1,283 @@
+  - proto/2023/wrapped/int64/invalid
+  - proto/2023/wrapped/int64/valid
+  - proto/2023/wrapped/string/invalid
+  - proto/2023/wrapped/string/valid
+  - proto/2023/wrapped/uint32/invalid
+  - proto/2023/wrapped/uint32/valid
+  - proto/2023/wrapped/uint64/invalid
+  - proto/2023/wrapped/uint64/valid
+  - proto2/bool/invalid
+  - proto2/bool/valid
+  - proto2/bytes/invalid
+  - proto2/bytes/valid
+  - proto2/double/invalid
+  - proto2/double/valid
+  - proto2/duration/invalid
+  - proto2/duration/valid
+  - proto2/enum/invalid
+  - proto2/enum/valid
+  - proto2/fixed32/invalid
+  - proto2/fixed32/valid
+  - proto2/fixed64/invalid
+  - proto2/fixed64/valid
+  - proto2/float/invalid
+  - proto2/float/valid
+  - proto2/int32/invalid
+  - proto2/int32/valid
+  - proto2/int64/invalid
+  - proto2/int64/valid
+  - proto2/predefined_and_custom/invalid
+  - proto2/predefined_and_custom/valid
+  - proto2/repeated/invalid
+  - proto2/repeated/valid
+  - proto2/repeated/wrapped/bool/invalid
+  - proto2/repeated/wrapped/bool/valid
+  - proto2/repeated/wrapped/bytes/invalid
+  - proto2/repeated/wrapped/bytes/valid
+  - proto2/repeated/wrapped/double/invalid
+  - proto2/repeated/wrapped/double/valid
+  - proto2/repeated/wrapped/float/invalid
+  - proto2/repeated/wrapped/float/valid
+  - proto2/repeated/wrapped/int32/invalid
+  - proto2/repeated/wrapped/int32/valid
+  - proto2/repeated/wrapped/int64/invalid
+  - proto2/repeated/wrapped/int64/valid
+  - proto2/repeated/wrapped/string/invalid
+  - proto2/repeated/wrapped/string/valid
+  - proto2/repeated/wrapped/uint32/invalid
+  - proto2/repeated/wrapped/uint32/valid
+  - proto2/repeated/wrapped/uint64/invalid
+  - proto2/repeated/wrapped/uint64/valid
+  - proto2/sfixed32/invalid
+  - proto2/sfixed32/valid
+  - proto2/sfixed64/invalid
+  - proto2/sfixed64/valid
+  - proto2/sint32/invalid
+  - proto2/sint32/valid
+  - proto2/sint64/invalid
+  - proto2/sint64/valid
+  - proto2/standard_predefined_custom/custom/invalid
+  - proto2/standard_predefined_custom/predefined/invalid
+  - proto2/standard_predefined_custom/standard/invalid
+  - proto2/standard_predefined_custom/valid
+  - proto2/string/invalid
+  - proto2/string/valid
+  - proto2/timestamp/invalid
+  - proto2/timestamp/valid
+  - proto2/uint32/invalid
+  - proto2/uint32/valid
+  - proto2/uint64/invalid
+  - proto2/uint64/valid
+  - proto2/wrapped/bool/invalid
+  - proto2/wrapped/bool/valid
+  - proto2/wrapped/bytes/invalid
+  - proto2/wrapped/bytes/valid
+  - proto2/wrapped/double/invalid
+  - proto2/wrapped/double/valid
+  - proto2/wrapped/float/invalid
+  - proto2/wrapped/float/valid
+  - proto2/wrapped/int32/invalid
+  - proto2/wrapped/int32/valid
+  - proto2/wrapped/int64/invalid
+  - proto2/wrapped/int64/valid
+  - proto2/wrapped/string/invalid
+  - proto2/wrapped/string/valid
+  - proto2/wrapped/uint32/invalid
+  - proto2/wrapped/uint32/valid
+  - proto2/wrapped/uint64/invalid
+  - proto2/wrapped/uint64/valid
+  - proto3/bool/invalid
```

Comment:
I'm most concerned about the `proto3` tests below this point since we don't support `proto2` or `edition 2023`.